src/Microsoft.TestPlatform.CrossPlatEngine/TestEngine.cs

```diff
@@ -153,6 +158,11 @@ public TestEngine() : this(TestRuntimeProviderManager.Instance, new ProcessHelpe
         requestData.MetricsCollection.Add(
             TelemetryDataConstants.ParallelEnabledDuringExecution,
             parallelLevel > 1 ? "True" : "False");
+        requestData.MetricsCollection.Add(
+            TelemetryDataConstants.TestSessionId,
+            testRunCriteria.TestSessionInfo != null
```

Comment:
ditto

Reply:
Will fix this in a subsequent PR as I don't want to go through the review cycle again.